### PR TITLE
#167733799 Remove refernces to mongoDB

### PR DESCRIPTION
**Pull request by codecell**
**What does this PR do?**
Removes pre-installed non-relational DB(mongoDB) packages and removes references to  them

**Description of Task to be completed?**
Use NPM to uninstall the redundant packages
remove all references to the mongoose packages

**How should this be manually tested?**
Check package.json file to see that the redundant packages are removed?

**Any background context you want to provide?**
Our team mate that set up sequelize already installed postgres 'pg' package , so i'll skip that

**What are the relevant pivotal tracker stories?**
#167733799

### CODE REVIEW

**Review by kingsley-einstein — COMMENTED** [review]
Good work mate!

File: src/routes/api/users.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-const mongoose = require("mongoose");
+// const mongoose = require("mongoose");
```
Comment (Oluwafayokemi) [inline src/routes/api/users.js:1]: Remove commented code

File: src/routes/api/users.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
 const router = require("express").Router();
 const passport = require("passport");
-const User = mongoose.model("User");
+// const User = mongoose.model("User");
```
Comment (Oluwafayokemi) [inline src/routes/api/users.js:4]: Remove commented code

File: src/index.js [hunk 3]
```diff
@@ -38,16 +38,15 @@ if (!isProduction) {
     app.use(errorhandler());
 }
 
-if (isProduction) {
-    mongoose.connect(process.env.MONGODB_URI);
-} else {
-    mongoose.connect("mongodb://localhost/conduit");
-    mongoose.set("debug", true);
-}
+// if (isProduction) {
```
Comment (Oluwafayokemi) [inline src/index.js:41]: Please, Remove all commented code

File: src/routes/api/users.js (hunk 2 above)
Comment (Ileriayo) [inline src/routes/api/users.js:4]: These comments don't serve any purpose and are therefore unnecessary, kindly remove them.

File: src/models/User.js [hunk 4]
```diff
@@ -1,57 +1 @@
...
-
-UserSchema.methods.toAuthJSON = function() {
-    return {
-        username: this.username,
-        email: this.email
-    };
-};
-
-mongoose.model("User", UserSchema);
+//use seqeulize schema
```
Comment (Ileriayo) [inline src/models/User.js:1]: No sequelize schema is implemented here. You might want to check that.

File: src/index.js [hunk 5]
```diff
@@ -38,16 +38,15 @@ if (!isProduction) {
...
+//     //remove mongoose conection string, use postgres db fr production
+// } else {
+//     //use postgres db for development
+// }
```
Comment (Ileriayo) [inline src/index.js:45]: These comments don't serve any purpose and are therefore unnecessary, kindly remove them.

**Review by Ileriayo — CHANGES_REQUESTED** [review]
Kindly check my review and effect changes. @codecell

File: src/index.js (hunk 5 above)
Comment (codecell) [reply src/index.js:45]: i'm on it
thanks

File: src/models/User.js (hunk 4 above)
Comment (codecell) [reply src/models/User.js:1]: yh, kingsley already did that, so i left tha blankt, once his PR is merged,  we'l make use of his own code

File: src/index.js (hunk 3 above)
Comment (codecell) [reply src/index.js:41]: ok, on it

**Pull request closed without merging**